doc/doxygen/public/TOPP.doxygen

```diff
@@ -139,6 +139,7 @@
   - @subpage TOPP_PeptideIndexer - Refreshes the protein references for all peptide hits.
   - @subpage TOPP_PhosphoScoring - Scores potential phosphorylation sites in order to localize the most probable sites.
   - @subpage TOPP_ProteinInference - Infer proteins from a list of (high-confidence) peptides.
+  - @subpage TOPP_PercolatorAdapter - Percolates the given protein/peptide identifications.
```

Comment:
Maybe better "Applies the percolator algorithm to protein/peptide identifications."